### DISCUSSION TITLE
“WinEdt Configuration Wizard” window always appears

### DISCUSSION THREAD

**Discussion by Hu-26** [WinEdt-Team/WinEdt-Support#76] [1.2 Bugs and Issues]
I installed WinEdt 11.2 but every time I open WinEdt, **the WinEdt Configuration Wizard window always appears !**
Even though I have left the WinEdt Configuration Wizard window unchecked "show this Wizard Next Time",  the WinEdt Configuration Wizard window still appears the next time I open WinEdt !

I can edit the tex file, or I can set the font size to 20 in the options and save it carefully. But when I start WinEdt again, my settings don't show up, even though the font size in the options has changed to 20 last time.
![1](https://github.com/user-attachments/assets/42986265-b0af-40bf-a212-1fb6a7b910c5)

I hope you can help me, thank you in advance!

**Comment by Alex-Simonic** [comment]
Click on the button "Browse Local Folder (%b)". Windows Explorer should open this folder. You should see files WinEdt.dnt and WinEdt.dnt.bak inside it. Note the time and date when these files were modified. Close WinEdt and refresh the contents of the folder in Windows Explorer (if necessary). If everything is OK WinEdt.dnt was renamed to WinEdt.dnt.bak (with the time of the previous WinEdt.dnt) and a new WinEdt.dnt should be created with the current time. 

Is that correct?

If so wait for a while to see if anything on your system removes or restores these files. Now start WinEdt again and check the time on these two files. WinEdt.dnt is the file where WinEdt stores all your changes (in a raw data format). Since your AppData folder appears OK external interference with WinEdt.dnt (perhaps by Anti Virus program) is the only reason for such behavior that I can think of. In any case, let us know if the date of WinEdt.dnt reflects changes as described above and we will see.

**Reply by Alex-Simonic** [reply]
I am having second thoughts about your application data folder. It should be something like:

`C:\Users\<user>\AppData\Roaming\WinEdt Team\WinEdt 11`

where `<user>` is your Windows user name. `Administrator.DESKTOP-3AREUII` is a strange user name, to say the least. Does this name change when you restart WinEdt?

Press `Ctrl+ALT+F9` and paste the following code as the command line:

`GetAppDataFolder(0,0);Prompt('%!0');`

Do you get the same folder (without \WinEdt Team\WinEdt 11)?

**Reply by Hu-26** [reply]
Thanks @Alex-Simonic！ I reinstalled it once and there is no more of the above problem, thank you again for your patience!
